WeatherXM/docs
# FE-1039 FE-1048 Fixes on images

## Includes
- Fix static images directories
- Fix images naming, use only lowercase with hyphens
- Remove not used images
- Fixed the size of images in "Rewards Claiming" and "How to join Arbitrum" pages

## How to test?
Branch Preview URL: https://feature-icon-names-fixes.docs-33x.pages.dev
1. Navigate throughout our docs and make sure all images are visible.
2. Ensure that the images in the "Rewards Claiming" and "How to join Arbitrum" look OK

## Other?
Removed "Video Instructions" section from D1, Helium and Pulse as these refer to M5 and feel out of context there.
